pom.xml

```diff
@@ -91,7 +91,7 @@
 
         <dep.airlift.version>247-SNAPSHOT</dep.airlift.version>
         <dep.bouncycastle.version>1.78</dep.bouncycastle.version>
-        <dep.bytebuddy.version>1.14.13</dep.bytebuddy.version>
+        <dep.bytebuddy.version>1.14.14</dep.bytebuddy.version>
```

Comment:
Can remove this property with Airbase 156

Reply:
Done.